### PR TITLE
PC-1295: Set up connection for open epc api

### PR DESCRIPTION
**Pull request by Glenn-Clarke**
[Link to Jira ticket](https://beisdigital.atlassian.net/browse/PC-1295)

# Description

Added the API Key and endpoint to the secrets template

Adding to feature branch as this would break dev without the following tickets.

# Checklist

- [x] I have made any necessary updates to the documentation
- [x] I have checked there are no unnecessary IDE warnings in this PR
- [x] I have checked there are no unintentional line ending changes
- [x] I have added tests where applicable
- [x] If I have made any changes to the code, I have used the IDE auto-formatter on it
- [x] If I have made any changes to Python files (even if not changing any content strings), I have run `make extract-translations`
- [x] If I have made any changes to website flow, I have updated the [Flow Miro Board](https://miro.com/app/board/uXjVK4AgQ3E=/)
- [x] If I have made any changes to website flow, I have checked forward and back behaviour is still consistent

### CODE REVIEW

File: envs/web.template
Comment (jdgage) [inline envs/web.template:1]: There's some other places these variables get used in actually allowing the code to use the API, which will be what the pipeline is unhappy about.

**Review by jdgage — CHANGES_REQUESTED** [review]

File: envs/web.template
Comment (Glenn-Clarke) [reply envs/web.template:1]: I think I've got them all, from some further searching 😅

File: help_to_heat/settings.py [hunk 1]
```diff
@@ -253,9 +253,8 @@ def show_toolbar(request):
 }
 
 OS_API_KEY = env.str("OS_API_KEY")
-EPC_API_BASE_URL = env.str("EPC_API_BASE_URL")
-EPC_API_CLIENT_ID = env.str("EPC_API_CLIENT_ID")
-EPC_API_CLIENT_SECRET = env.str("EPC_API_CLIENT_SECRET")
```
Comment (jdgage) [inline help_to_heat/settings.py:258]: Were these being used somewhere in the app? I'd expect them to be consumed somewhere we're actually using the API

Comment (Glenn-Clarke) [reply help_to_heat/settings.py:258]: Ah I wasn't sure that was part of the scope. I've made some further changes now.
I'm going to sort the tests, but could you give the code a skim? (understand time is tight today)

**Review by jdgage — APPROVED** [review]

File: help_to_heat/settings.py (hunk 1 above)
Comment (jdgage) [reply help_to_heat/settings.py:258]: Might not have been, but I suppose the auth part is useful to have, and it's only going to save future effort for the other bits. If it turns out to be a load of work or you're getting too close to actually doing the implementation then I suppose you could leave the current implementation as-is and have the new stuff commented out to be used on the implementation tickets?

**Pull request merged**